Examples/Scripts/Python/itk_seeding.py

```diff
@@ -3,18 +3,106 @@
 import argparse
 import tempfile
 import argparse
+import math
 
 import acts
 import acts.examples
 
 from acts.examples import CsvSpacePointReader
+from collections import namedtuple
+from acts.examples.reconstruction import (
+    SeedfinderConfigArg,
+    SeedFilterConfigArg,
+    SpacePointGridConfigArg,
+    SeedingAlgorithmConfigArg,
+)
 
 u = acts.UnitConstants
 
 
-def runITkSeeding(field, inputSPs, outputDir, inputSpacePointsType, s=None):
+def itkSeedingAlgConfig(inputSpacePointsType):
```

Comment:
Can you move `itkSeedingAlgConfig()` to `Examples/Python/python/acts/examples/itk.py`?
(I propose, not `reconstructon.py`)

Reply:
done!